src/Mod/Fem/Gui/CMakeLists.txt

```diff
@@ -428,7 +428,7 @@ SET(FemGuiPythonUI_SRCS
     Resources/ui/MeshRegion.ui
     Resources/ui/ResultHints.ui
     Resources/ui/ResultShow.ui
-    Resources/ui/SolverCalculix.ui
+    Resources/ui/SolverCalculiX.ui
     Resources/ui/SolverCalculiX.ui
```

Comment:
if the intention was really to replace the old one then this duplication should be removed